tests/test_listen_addr.py

```diff
@@ -36,11 +36,11 @@ def get_value(self):
 
 def run(party, is_inner_party):
     compatible_utils.init_ray(address='local')
-    cluster = {
-        'alice': {'address': '127.0.0.1:11012', 'listen_addr': '0.0.0.0:11012'},
```

Comment:
As a result, `addresses` still can't contain just the address of party, but a dict where `listening_on` fields (and  others maybe) also supported

Reply:
We could add a simple helper here and support both `str` and `dict` as addresses value, if you think `listen_addr` and `address` are the same in common cases